test/test_observation_poisson.py

```diff
@@ -19,4 +19,4 @@ def test_poisson_obs():
     with numpyro.handlers.seed(rng_seed=223):
         sim_pois, *_ = pois(mu=rates)
```

Comment:
This can and should now be this, yes? 

```suggestion
        sim_pois = pois(mu=rates)
```

Reply:
addressed in https://github.com/CDCgov/PyRenew/pull/441/commits/3206d053b132e445a797f29fb82148bc7fe43674